components/sections/Even.jsx

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+import React from "react";
+import { Span } from "../ui/Typography";
+
+export default function Even() {
+  return (
+    <div className="mt-17.5 sm:mt-44.5 xl:mt-64.5">
```

Comment:
Remove the classes for margin-top since they are being implemented in the header.